db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_07_04_151452) do
+ActiveRecord::Schema.define(version: 2021_07_11_152754) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "events", force: :cascade do |t|
```

Comment:
does this need to be in a migration?

Reply:
No, the schema is where the final database structure lives. There is a create table migration.